operations/workplace/README.md

```diff
@@ -4,31 +4,31 @@ description: Description of norms on Mattermost Community server
 
 # Workplace
 
-## What is the Mattermost pre-release server?
+## What is the Mattermost Pre-release Server?
```

Comment:
I'm not 100% sure if it makes sense to call it pre-release server since it is now called community server. Not sure who would know what the norm these days is for the name of the server.

Reply:
I meant to comment on that and I agree. Let's go with community seeing as that's the link name, and what we commonly refer to?